cfpb/cfgov-refresh
# Fix for askCFPB Spanish questions and css file generation

Fix for askCFPB Spanish questions and css file generation

## Removals

- Removed `cfgov/legacy/static/knowledgebase/es-ask-styles-ie.min.css` so the new minified file will get picked up by the asset pipeline.

- Removed `cfgov/legacy/static/knowledgebase/es-ask-styles.min.css` so the new minified file will get picked up by the asset pipeline.

## Changes

- Modified `cfgov/legacy/static/knowledgebase/less/es-ask-base.less`  to bring it inline with minified file.
- Modified  `gulp/tasks/styles.js` to generate styles for `knowledgebase`.

## Testing

- Run `gulp styles`
- Visit `http://localhost:8000/es/obtener-respuestas/c/enviar-dinero/1507/cuales-son-las-nuevas-protecciones-federales-a-los.html`
- Observe that the page is correctly rendered.
- Inspect the console to ensure no 404 errors are generated.

## Screenshots

<img width="737" alt="screen shot 2017-01-18 at 10 14 38 am" src="https://cloud.githubusercontent.com/assets/1696212/22069588/f303f8d4-dd66-11e6-9c87-f63c9465616f.png">


## Checklist
